tests/test_dispatcher.py

```diff
@@ -730,6 +730,37 @@ def test_sensible_worker_thread_names(self, dp2):
         for thread_name in thread_names:
             assert thread_name.startswith(f"Bot:{dp2.bot.id}:worker:")
 
+    @pytest.mark.parametrize(
+        'message',
+        [
+            Message(message_id=1, chat=Chat(id=1, type=None), migrate_from_chat_id=1, date=None),
+            Message(message_id=1, chat=Chat(id=1, type=None), migrate_to_chat_id=1, date=None),
+            Message(message_id=1, chat=Chat(id=1, type=None), date=None),
+        ],
+    )
+    @pytest.mark.parametrize(
+        'old_chat_id,new_chat_id', [(None, None), (1, None), (None, 1), (1, 1)]
+    )
+    def test_migrate_chat_data(self, message: 'Message', old_chat_id: int, new_chat_id: int):
+        if (message and (old_chat_id or new_chat_id)) or not any(
+            (message, old_chat_id, new_chat_id)
+        ):
+            with pytest.raises(ValueError) as exc:
+                Dispatcher.migrate_chat_data(None, message, old_chat_id, new_chat_id)
```

Comment:
```suggestion
                dp.migrate_chat_data(message, old_chat_id, new_chat_id)
```
`Dispatcher` is a class - but we need an instance of that class ;) the `dp` fixture provides such an instance for you

Reply:
Ops, I have just see your suggestions 😅